Tweaks to visbility of innards

comma opened the pull request:
Now you can see bones when you got retracted-sized wounds.
You can also gleam some information about organs you see inside.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

Techhead0 commented on code/modules/organs/internal/_internal.dm:
```diff
@@ -135,4 +135,20 @@
+		. = "bits and pieces of a destroyed "
+	else if(is_broken())
+		. = "broken "
+	else if(is_broken())
```
Meant is_bruised() or whatever it's supposed to be??

comma replied:
derp

PurpleMartinJCK commented:
Is this because of the round I kept grabbing people to check their organs?

comma commented:
Nah, I had 'make necrotic organs visible' in todo for weeks.

comma committed:
Tweaks to visbility of innards

Now you can see bones when you got retracted-sized wounds.
You can also gleam some information about organs you see inside.
